src/Nethermind/Nethermind.Merge.AuRa.Test/AuRaMergeEngineModuleTests.cs

```diff
@@ -87,14 +99,138 @@ public override Task Can_apply_withdrawals_correctly((Withdrawal[][] Withdrawals
         return base.Can_apply_withdrawals_correctly(input);
     }
 
+    private async Task<ValueTuple<int, Transaction[]>> ProduceBlockWithTransactions(IEngineRpcModule rpc, MergeTestBlockchain chain, uint count)
+    {
+        int id = _blocksProduced;
+
+        _parentPayload!.TryGetBlock(out Block? parentBlock);
+        chain.AddTransactions(BuildTransactions(chain, parentBlock!.CalculateHash(), TestItem.PrivateKeyC, TestItem.AddressF, count, id, out _, out _));
+
+        IReadOnlyList<ExecutionPayload> executionPayloads = await ProduceBranchV1(rpc, chain, 1, _parentPayload, true);
+        executionPayloads.Should().HaveCount(1);
+
+        _parentPayload = executionPayloads[0];
+        _blocksProduced++;
+
+        return (id, _parentPayload.GetTransactions());
+    }
+
+    [Test]
+    public async Task Can_include_shutter_transactions()
+    {
+        using MergeTestBlockchain chain = await new MergeAuRaTestBlockchain(null, null, true).Build(new TestSingleReleaseSpecProvider(London.Instance));
+        IEngineRpcModule rpc = CreateEngineModule(chain);
+
+        // creating chain with 3 blocks
+        IReadOnlyList<ExecutionPayload> executionPayloads = await ProduceBranchV1(rpc, chain, 3, CreateParentBlockRequestOnHead(chain.BlockTree), true);
+        _parentPayload = executionPayloads.Last();
+
+        // should contain shutter transactions
+        executionPayloads[0].GetTransactions().Should().HaveCount(1);
+        executionPayloads[0].GetTransactions()[0].Value.Should().Be(123);
+
+        executionPayloads[1].GetTransactions().Should().HaveCount(1);
+        executionPayloads[1].GetTransactions()[0].Value.Should().Be(124);
+
+        executionPayloads[2].GetTransactions().Should().HaveCount(1);
+        executionPayloads[2].GetTransactions()[0].Value.Should().Be(125);
+
+        // build blocks with transactions from both shutter and TxPool
+        // id is used as value to differentiate between transactions
+        (int id0, Transaction[] transactions0) = await ProduceBlockWithTransactions(rpc, chain, 3);
+        transactions0.Should().HaveCount(4);
+        transactions0[0].Value.Should().Be(126);
+        transactions0[1].Value.Should().Be(id0.GWei());
+        transactions0[2].Value.Should().Be(id0.GWei());
+        transactions0[3].Value.Should().Be(id0.GWei());
```

Comment:
this should be 1-liner:
`transactions.Select(t => t.Value).Should().BeEquivalentTo(new[]{ 126, id0.Gwei(), id0.Gwei(),id0.Gwei()})`

Reply:
I'm going to do some improvements to the unit tests, this one might be removed